en-US/ranking/criteria.md

```diff
@@ -46,70 +78,106 @@ The following requirements are for the individual maps themselves.
 * **The maps must be at least 45 seconds long.**
 * **The maps must be timed as accurately as possible.** Timing points must not be used in place of scroll velocity changes.
 * **The maps must have a song select preview point that sufficently compliments the song.** It's usually placed before the chorus or at the beginning of a section. A solid preview point attracts more players to try the map.
+* **BPM lines should not be used abusively for decorations or color changes.** This could be a confusing to players to play and redundant as in filesize.
```

Comment:
```suggestion
```

Reply:
Or move to guidelines:

"Timing points should properly reflect the song. When used improperly, they can cause confusion to the player. "

Reply:
moved is ok